init.lua

```diff
@@ -108,3 +108,25 @@ minetest.register_on_joinplayer(function(player)
 		hide_player(player)
 	end, player:get_player_name())
 end)
+
+-- /whereis chat-command
+minetest.register_chatcommand("whereis", {
+	params = "<name>",
+	description = "Get location of player",
+	privs = { kick = true, ban = true, fly = true },
```

Comment:
May I ask why the fly priv is required?

Moderators don't have the fly priv, as far as I know.